Python/Product/Analysis/ModuleTable.cs

```diff
@@ -159,7 +159,12 @@ class ModuleTable : IEnumerable<KeyValuePair<string, ModuleLoadState>> {
 
                 var builtinModule = moduleRef.Module as BuiltinModule;
                 if (builtinModule != null) {
-                    if (!newNames.Contains(name)) {
+                    IPythonModule newModule = null;
+                    if (newNames.Contains(name)) {
+                        newModule = _interpreter.ImportModule(name);
+                    }
+
+                    if (newModule == null) {
```

Comment:
This section is the fix. Most of the rest of the changes are to allow testing `ModuleTable` without setting up a full analyzer.